.github/workflows/ci.yml

```diff
@@ -67,13 +67,19 @@ jobs:
           exit-code: 1
           severity: CRITICAL
           format: table
+      - name: Define IMAGE_TAG (master)
+        if: ${{ github.ref == 'refs/heads/master' }}
+        run: echo "IMAGE_TAG=latest-${{ matrix.architecture }}" >> $GITHUB_ENV
+      - name: Define IMAGE_TAG (tags)
+        if: ${{ github.ref != 'refs/heads/master' && (contains(github.ref, 'tags') && contains(github.ref, matrix.architecture)) }}
```

Comment:
This can be simplified.  `github.ref != 'refs/heads/master` is redundant since it will never be false if the following check is true.